MapboxVision/src/test/java/com/mapbox/vision/telemetry/TotalBytesCounterTest.kt

```diff
@@ -0,0 +1,274 @@
+package com.mapbox.vision.telemetry
+
+import com.mapbox.vision.utils.system.Time
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.*
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.gherkin.Feature
+import java.util.concurrent.TimeUnit
+
+const val DEFAULT_SESSION_MAX_BYTES = 10_000L
+val DEFAULT_SESSION_LENGTH_MS = TimeUnit.MINUTES.toMillis(10)
+val CURRENT_TIME = TimeUnit.HOURS.toMillis(20)
+
+
+object TotalBytesCounterTest : Spek({
+    Feature("TotalBytesCounter") {
+
+        Scenario("Check default session size") {
+            lateinit var totalBytesCounter: TotalBytesCounter
+            val mockedTime = mockk<Time>()
+
+            Given("TotalBytesCounter with default params") {
+                totalBytesCounter = TotalBytesCounter.Impl(time = mockedTime)
+            }
+
+            var actualValue = false
+
+            When("Get fitInLimit for 30MB") {
+                actualValue = totalBytesCounter.fitInLimit(30 * 1024 * 1024L)
```

Comment:
maybe save 30 Mb as a constant. So if we change it, it will be updated in all params and messages

Reply:
Good idea